specification/dimensions/skupriceid.md

```diff
@@ -0,0 +1,36 @@
+### SKU Price ID
+
+A SKU Price ID is a unique identifier that defines the price that is used to bill the SKU of the charge. This identifier represents a unique collection of nullable attributes that determine the price. SKU Price ID can be referenced on a price list published by a provider to look up detailed information about the SKU and the unit price associated with the SKU. The composition of the detailed information associated with the price may differ from one provider to the next.
```

Comment:
```suggestion
A SKU Price ID is a unique identifier that defines the unit price associated with the SKU of the charge. This identifier represents a unique collection of nullable attributes that determine the price. SKU Price ID can be referenced on a price list published by a provider to look up detailed information about the SKU and the unit price associated with the SKU. The composition of the detailed information associated with the price may differ from one provider to the next.
```

Reply:
>  that defines the unit price associated with the SKU of the charge

It looks to me with this formulation that a SKU has a single unit price, which is not true.

Instead, the formulation "that defines the price that is used to bill the SKU of the charge" highlights that a unit price has been selected to bill the SKU.